conan/test_package/test_package.cpp

```diff
@@ -0,0 +1,68 @@
+#include "behaviortree_cpp/behavior_tree.h"
+#include "behaviortree_cpp/bt_factory.h"
+
+using namespace BT;
+
+NodeStatus SayHello()
+{
+    printf("hello\n");
```

Comment:
why do you mix printf and cout in this example?

Reply:
it's from https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/examples/broken_sequence.cpp